File: examples/user/updatePassword/node/updateTheAccountPassword.js
const { UserService } = require("m3o/user");

// Update the account password
async function updateTheAccountPassword() {
  let userService = new UserService(process.env.MICRO_API_TOKEN);
  let rsp = await userService.updatePassword({
    confirmPassword: "Password2",
    id: "user-1",
    newPassword: "Password2",
    oldPassword: "Password1",
  });
  console.log(rsp);
}

updateTheAccountPassword();

File: examples/user/updatePassword/go/updateTheAccountPassword.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/user"
)

// Update the account password
func UpdateTheAccountPassword() {
	userService := user.NewUserService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := userService.UpdatePassword(&user.UpdatePasswordRequest{
		ConfirmPassword: "Password2",
		NewPassword:     "Password2",
		OldPassword:     "Password1",
	})
	fmt.Println(rsp, err)
}
